tests/test_simpleButler.py

```diff
@@ -794,6 +798,85 @@ def makeButler(self, writeable: bool = False) -> Butler:
         DatastoreMock.apply(butler)
         return butler
 
+    def test_region_queries(self):
```

Comment:
The rest of the query tests are in `lsst/daf/butler/tests/butler_queries.py`.  That might be a better place for these because it runs against all database configurations (sqlite, postgres, remote).  `test_simplebutler.py` doesn't cover postgres.

Reply:
Thanks for pointing that out. I moved them.